File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Types/Inheritance.cs
using System;
using System.Collections.Generic;

#region Class

namespace Class
{
    public class InheritType : Exception
    {
    }

    public class ImplementInterfaceType : IDisposable
    {
    }

    public class MixedInheritanceType : Exception, IDisposable
    {
    }
}

#endregion

#region Struct

namespace Struct
{
    public struct ImplementInterfaceType : IDisposable
    {
    }
}

#endregion

#region Interface

namespace Interface
{
    public interface ImplementInterfaceType : IDisposable
    {
    }
}

#endregion

#region Enum

namespace Enum
{
    public enum InheritPrimitiveType : Int32
    {
    }
}

#endregion

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Types/Inheritance.vb
Imports System
Imports System.Collections.Generic

#Region "Class"

Namespace Class
    Public Class InheritType
        Inherits Exception
    End Class

    Public Class ImplementInterfaceType
        Implements IDisposable
    End Class

    Public Class MixedInheritanceType
        Inherits Exception
        Implements IDisposable
    End Class
End Namespace

#End Region

#Region "Struct"

Namespace Struct
    Public Structure ImplementInterfaceType
        Implements IDisposable
    End Structure
End Namespace

#End Region

#Region "Interface"

Namespace Interface
    Public Interface ImplementInterfaceType
        Inherits IDisposable
    End Interface
End Namespace

#End Region

#Region "Enum"

Namespace Enum
    Public Enum InheritPrimitiveType As Int32
    End Enum
End Namespace

#End Region